pyCGM_Single/pycgmCalc.py

```diff
@@ -47,24 +47,55 @@ def calcKinetics(data, Bodymass):
     
 
 def calcAngles(data,**kargs):
+    """Calculates the joint angles and axis
+    
+    By default, the function will calculate all the data and return angles
+    and axis as separate arrays
+
+    Parameters
+    ----------
+    data : list of dict
+        Joint centres in the global coordinate system. List indices correspond 
+        to each frame of trial. Dict keys correspond to name of each joint centre,
+        dict values are arrays ([],[],[]) of x,y,z coordinates for each joint 
+        centre
+    **kargs : dict of keyword arguments
+        start
+           Indicates which index in `data` to start the calculation
+        end
+           Indicates which index in `data` to end the calculation
+        frame : int
+            Frame number if the calculation is only for one frame
+        cores : int
+            Number of processes to use on the calculation
+        vsk : dict
+            Vsk file as a dictionary or label and data
+        angles : boolean
+            If true, the function will return the angles
+        axis : boolean
+            If true, the function will return the axis
+        splitAnglesAxis : boolean
+            If true, the function will return the angles and axis as
+            separate arrays. If false, it will be the same array
+        multiprocessing : boolean
+            If true, the function will use multiprocessing
+    
+    Returns
+    -------
+        r : list of list
```

Comment:
Double check this, I'm not sure Returns can be formatted like this (and I don't think its true). Return can be one value, but that value can be a list or tuple, so thats how it should be shown